defaults/main.yml

```diff
@@ -39,6 +39,9 @@ archivematica_src_am_version: "qa/1.x"
 archivematica_src_ss_version: "qa/0.x"
 archivematica_src_devtools_version: "master"
 
+archivematica_src_am_env_django_secret_key: "CHANGE_ME_WITH_A_SECRET_KEY"
+archivematica_src_am_env_django_allowed_hosts: "127.0.0.1"
```

Comment:
Should the value here not be "*"? That's what the value of the equivalent constant in src/dashboard/src/settings/common.py used to be. I tested an AM deploy using this branch for the AM ansible role along with the dev/clean-up-config branch of AM. If I set `archivematica_src_am_env_django_allowed_hosts` to `192.168.168.192` then AIP re-ingest would fail. If I left it at its default `127.0.0.1` then I would get a 500 error when starting the application: ``ProgrammingError: (1146, "Table 'MCP.django_session' doesn't exist")``